### DISCUSSION TITLE
Space between lines

### DISCUSSION THREAD

**Discussion by Mopsior** [JanDeDobbeleer/oh-my-posh#2022] [Troubleshoot]
Hi, could I create some space between lines? I'm using modified bubblesline theme.
![image](https://user-images.githubusercontent.com/80514290/161379960-29ee0b2f-76c5-4daa-99dc-7ca7bac55c52.png)

**Comment by JanDeDobbeleer** [comment]
What space do you need? The block on the right first, newline, then the left block?

**Reply by Mopsior** [reply]
Little amount of space, like little Margin on top of line

**Reply by JanDeDobbeleer** [reply]
It's not css, so the only thing you can do is start on a newline using the newline property on [block](https://ohmyposh.dev/docs/config-block).

**Reply by ghost** [reply]
> It's not css, so the only thing you can do is start on a newline using the newline property on [block](https://ohmyposh.dev/docs/config-block).
I would also be interest in knowing how to increase the linespacing between consecutive inputs in the terminal [WSL with Ubuntu 20.04]. Could you elaborate on the newline property?

**Reply by JanDeDobbeleer** [reply]
@tpjerry right this way: https://ohmyposh.dev/docs/configuration/block

**Reply by ghost** [reply]
OK! Noted! I have to do: `"newline", true`. But now it gets too big. I will live with it. Thanks for the help!

**Reply by JanDeDobbeleer** [reply]
@tpjerry oh-my-posh can't adjust the line height, that's a terminal feature.

**Reply by budaesandrei** [reply]
> @tpjerry oh-my-posh can't adjust the line height, that's a terminal feature.

I tried the line height in terminal but that heightens the oh-my-posh blocks too...I guess it would be great to have some padding control instead
